C#/number-sort.cs

```diff
@@ -2,7 +2,7 @@ public class Sort
 {
     public static void Main()
     {
-        int[] myArr= new int[5] {1,5,3,8,21};
+        int[] myArr= new int[5] {1,9,728,320,123,845,21};
```

Comment:
You've initialized the size of the array as `5`, but the number of elements in the array are more than five, it breaks the program.

Reply:
Oh